app/models/induction_record.rb

```diff
@@ -138,4 +139,10 @@ def transferred?
   def update_analytics
     Analytics::UpsertECFInductionJob.perform_later(induction_record: self) if saved_changes?
   end
+
+  def validate_sensible_dates
```

Comment:
It would be really good to understand why this pattern happens and address that scenario (not in this PR).